src/storage/txn/commands.rs

```diff
@@ -321,6 +365,11 @@ command! {
         }
 }
 
+impl CommandExt for Prewrite {
```

Comment:
Just a syntactic nit, but do you think we could move these inside the `command` macro use?

Reply:
> Just a syntactic nit, but do you think we could move these inside the `command` macro use?

I’ve tried it, but things become tricky when handleing `self `.